data/binding/sprintf.go

```diff
@@ -203,3 +203,16 @@ func (s *sprintfString) Set(str string) error {
 
 	return nil
 }
+
+// StringToStringWithFormat creates a binding that converts a string to another string using the specified format.
+// Changes to the returned String will be pushed to the passed in String and setting a new string value will parse and
+// set the underlying String if it matches the format and the parse was successful.
+//
+// Since: 2.2
+func StringToStringWithFormat(str String, format string) String {
+	if format == "%s" { // Same as not using custom formatting.
+		return str
+	}
```

Comment:
Might it be worth to also test this part in the tests that you added? It should probably even out the coverage.

Reply:
Good point, thanks!